tree/ntupleutil/v7/test/ntuple_inspector.cxx

```diff
@@ -135,17 +135,25 @@ TEST(RNTupleInspector, SizeCompressedInt)
       writeOptions.SetCompression(505);
       auto ntuple = RNTupleWriter::Recreate(std::move(model), "ntuple", fileGuard.GetPath(), writeOptions);
 
-      for (int32_t i = 0; i < 50; ++i) {
+      for (int32_t i = 0; i < 500; ++i) {
          *nFldInt = i;
          ntuple->Fill();
+
+         // Store the data in ten clusters to be able to test that the size is correctly computed in this way.
+         if (i % 50 == 49)
+            ntuple->CommitCluster();
       }
    }
 
    auto inspector = RNTupleInspector::Create("ntuple", fileGuard.GetPath());
 
-   EXPECT_EQ(sizeof(int32_t) * 50, inspector->GetUncompressedSize());
-   EXPECT_GT(inspector->GetCompressedSize(), 0);
+   EXPECT_EQ(sizeof(int32_t) * 500, inspector->GetUncompressedSize());
    EXPECT_LT(inspector->GetCompressedSize(), inspector->GetUncompressedSize());
+   // Check the target size with a 5% tolerance to account for small fluctuations across different platforms.
+   std::uint64_t targetSize = 800;
+   EXPECT_GT(inspector->GetCompressedSize(), targetSize - targetSize * .05f);
+   EXPECT_LT(inspector->GetCompressedSize(), targetSize + targetSize * .05f);
```

Comment:
Again nitpick, but maybe another way to express this in one line of code should be
```suggestion
   EXPECT_NEAR(inspector->GetCompressedSize(), targetSize, targetSize * .05f);
```

Reply:
Ah neat, I didn't know this existed!